boot/zephyr/socs/nrf54h20_cpuapp.conf

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Configuration which is needed for supporting resume the application execution
+# after suspend to RAM (S2RAM) requested by the application.
+# MCUboot does not support S2RAM itself, but serves as an immediate actor while waking up
+# from suspension.
+CONFIG_PM=y
+CONFIG_PM_S2RAM=y
+CONFIG_PM_S2RAM_CUSTOM_MAKING=y
```

Comment:
Shouldn't it be "CONFIG_PM_S2RAM_CUSTOM_MARKING" instead of "CONFIG_PM_S2RAM_CUSTOM_MAKING" (missing "R")?